### PR TITLE
feat:add solution for day 14

### PR DESCRIPTION
**Pull request by Huynhtram1504**
## Summary by CodeRabbit

- **New Features**
	- Introduced a BMI calculation function that categorizes body types based on height and weight.
	- Added a registration page test to verify user registration functionality.

- **Bug Fixes**
	- Updated Playwright dependency to ensure compatibility and improvements.

- **Tests**
	- Added new test cases to validate page title and "Get started" link functionality.
	- Removed comprehensive test suite for the Todo application.

- **Chores**
	- Updated `.gitignore` to exclude sensitive authentication files.

### CODE REVIEW

**Review by minhphong306 — CHANGES_REQUESTED** [review]
Please fix the comments

File: daily-challenges/2024-09/14/huynhtram1504.spec.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { test, expect } from '@playwright/test';
+//Javascript
+//Write function to calculate BMI
+function calculateBMI(height, weight){
+    const BMI = weight/(height*height);
+    let bodyType = '';
+    if(BMI< 18.5){
```
[the remark this answers is not in the document]
Comment (Huynhtram1504) [reply daily-challenges/2024-09/14/huynhtram1504.spec.ts:7]: You mean that format code line 4 as  "function calculateBMI(height: number, weight" number){"

[the remark this answers is not in the document]
Comment (playwrightvietnam) [reply daily-challenges/2024-09/14/huynhtram1504.spec.ts:7]: No, the spacing. I think your code is not formatted, so the spacing look weird
```suggestion
    if(BMI < 18.5){
```

**Comment by minhphong306** [comment]
Please remove all unrelated changes, @Huynhtram1504 
I mean, your pull request should only contain the changes in the file: [daily-challenges/2024-09/14/huynhtram1504.spec.ts](https://github.com/playwrightvn/pw-discovery/pull/191/files/d01eca4abad005583544c898d2ec73014390c357#diff-cbe56c1d2b1dbb4f9aa9e186795bf9d2fa37bd1c42cdbe915350a8cdee164069)

**Comment by minhphong306** [comment]
Please update when you re-open it.
I need to close this PR because I want to track only the active PRs with in three days.

**Pull request closed without merging**